src/zui/ZUIOrganizeSidebar.tsx

```diff
@@ -1,219 +1,222 @@
+    { icon: <Event />, name: 'projects' },
+  ] as const;
+
+  const OrganizeSideBar = (
+    <Box sx={{ display: 'flex' }}>
+      <CssBaseline />
+      <Drawer
+        onFocus={() => void 0}
+        onMouseLeave={() => {
+          setHover(false);
+        }}
+        onMouseOver={() => {
+          setHover(true);
+        }}
+        open={open}
+        sx={{
+          alignItems: open ? 'flex-start' : 'center',
+        }}
+        variant="permanent"
+      >
+        <DrawerHeader>
+          <IconButton onClick={toggleDrawer}>
+            {!open && hover && !expanded && (
+              <KeyboardDoubleArrowRightIcon
+                onClick={toggleDrawer}
+                sx={{ margin: 1 }}
+              />
+            )}
+            {!open && !hover && !expanded && (
+              <NextLink href="/organize" passHref>
+                <Avatar
+                  alt="icon"
+                  src={`/api/orgs/${orgId}/avatar`}
+                  sx={{ alignSelf: 'center' }}
+                />
+              </NextLink>
+            )}
+            {expanded && (
+              <ZUIFuture future={model.getOrganization(orgId)}>
+                {(data) => {
+                  return (
+                    <Box alignSelf="center">
+                      <Box display="flex" justifyContent="start">
+                        {hover ? (
+                          <ChevronLeftIcon
+                            onClick={handleDrawerToggle}
+                            sx={{ alignSelf: 'center', margin: 2 }}
+                          />
+                        ) : (
+                          <Avatar
+                            alt="icon"
+                            src={`/api/orgs/${orgId}/avatar`}
+                            sx={{ alignSelf: 'center', marginLeft: 2 }}
+                          />
+                        )}
+                        <Typography gutterBottom m={2} variant="h6">
+                          {data.title}
+                        </Typography>
+                      </Box>
+                    </Box>
+                  );
+                }}
+              </ZUIFuture>
+            )}
+          </IconButton>
+        </DrawerHeader>
+        <Divider />
+        <List>
+          {menuItemsMap.map((item) => (
+            <ListItem
+              key={item.name}
+              disableGutters
+              disablePadding
+              sx={{ display: 'block' }}
+            >
+              <NextLink
+                href={
+                  item.name === 'home'
+                    ? '/organize/'
+                    : `/organize/${orgId}/${item.name}`
+                }
+                passHref
               >
-                <Event />
-              </IconButton>
-            </NextLink>
-          </ListItem>
-        </Box>
-      </List>
+                <IconButton
+                  color={
+                    key.startsWith('/' + item.name) ? 'primary' : 'secondary'
```

Comment:
These are not the correct states, unless I'm mistaken. This code will render the "selected" icon in red (`'primary'`) but should really, according to the design, render the icon with the same color, but with a slightly tinted background. 

![image](https://github.com/zetkin/app.zetkin.org/assets/550212/0c0f2da3-1938-4d1d-a5c0-0e1808faf2e5)